### PR TITLE
completed es6-operators

### PR DESCRIPTION
**Pull request by t-doan**
es6-operators

### CODE REVIEW

File: es6-operators/main.js [hunk 1]
```diff
@@ -19,10 +19,10 @@ console.log('v2:', v2 && 'v2 is truthy');
...
  * Expected: `config1 = { value: 'Something' }`, `config2 = 'default-value'`.
  */
-const config1 = 'default-value';
-console.log('config1:', v1 || config1);
-const config2 = 'default-value';
-console.log('config2:', v2 || config2);
+const config1 = v1 || 'default-value';
+console.log('config1:', config1);
+const config2 = v2 || 'default-value';
+console.log('config2:', config2);
```
Comment (t-doan) [inline es6-operators/main.js:25]: assigned the conditional to the variable

File: es6-operators/main.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+const v4 = [3, 5, 7, 9];
+
+/* TODO:
+ * Demonstrate how to execute an expression only if a variable is truthy.
+ * In one expression, log "v1 is truthy" to the console only if `v1` is truthy.
+ * In one expression, log "v2 is truthy" to the console only if `v2` is truthy.
+ * Expected: v1 message logs, v2 message does not.
+ */
+console.log('v1:', v1 && 'v1 is truthy');
+console.log('v2:', v2 && 'v2 is truthy');
```
Comment (shawnkost) [inline es6-operators/main.js:14]: Not quite! This is technically still logging regardless. The `console.log` should only fire if `v1` or `v2` evaluates to truthy

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here Tai!

File: es6-operators/main.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+const v4 = [3, 5, 7, 9];
+
+/* TODO:
+ * Demonstrate how to execute an expression only if a variable is truthy.
+ * In one expression, log "v1 is truthy" to the console only if `v1` is truthy.
+ * In one expression, log "v2 is truthy" to the console only if `v2` is truthy.
+ * Expected: v1 message logs, v2 message does not.
+ */
+if (v1 && v1) console.log('v1 is truthy');
+if (v2 && v2) console.log('v2 is truthy');
```
Comment (shawnkost) [inline es6-operators/main.js:14]: No `if` statements

**Review by shawnkost — CHANGES_REQUESTED** [review]
Another change here!

File: es6-operators/main.js [hunk 4]
```diff
@@ -10,8 +10,8 @@ const v4 = [3, 5, 7, 9];
  * In one expression, log "v2 is truthy" to the console only if `v2` is truthy.
  * Expected: v1 message logs, v2 message does not.
  */
-if (v1 && v1) console.log('v1 is truthy');
-if (v2 && v2) console.log('v2 is truthy');
+v1 && console.log('v1 is truthy');
+v2 && console.log('v2 is truthy');
```
Comment (t-doan) [inline es6-operators/main.js:14]: changed to this

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**